src/App.js

```diff
@@ -1,29 +1,24 @@
 import React from 'react';
-
+import PostList from './components/PostList';
 import './App.scss';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+const preparedPosts = posts.map(post => (
+  {
+    ...post,
+    user: users.find(user => post.userId === user.id),
+    comment: comments.find(comment => post.id === comment.postId),
+  }
+));
+
```

Comment:
I think new obj is redundant. But do as you think

Reply:
You add only one comment, but we have more!!!  https://prnt.sc/132x364
Use filter insteadfind